lib/controllers/CustomerController.ts

```diff
@@ -7,8 +7,17 @@ import CustomerNotFoundException from "../exceptions/CustomerNotFoundException";
 
 export class CustomerController{
 
+    private static controller : CustomerController;
     private model: CustomerModel;
-    constructor(){
+
+    public static getInstance = () => {
```

Comment:
wenn du hier über die static die methode aufrufst kann es sein das this.model noch nicht gesetzt ist. Ist das gewollt?

Reply:
Öhm okay, wie denn? Ich stell doch sicher, dass ich eine valide instanz zurückgebe und diese hat das model dann im constructor gesetzt

Reply:
Ja aber du rufst doch die Methode static auf. Das bedeutet es wird ja keine neue Instance der Klasse erstellt. Und der Constructor daher wiederum nicht aufgerufen.

Reply:
ne vergiss es. Mein fehler :D.